test/onnx_import.cpp

```diff
@@ -112,3 +113,33 @@ TEST(onnx, model_split_variable_parts_2d)
         EXPECT_EQ(expected_output[i], result_vectors[0]);
     }
 }
+
+TEST(onnx, model_batchnorm_default)
+{
+    // Batch Normalization with default parameters
+    auto function{ngraph::onnx_import::import_onnx_function(
+        ngraph::file_util::path_join(SERIALIZED_ZOO, "onnx/batchnorm_default.onnx"))};
+
+    std::vector<std::vector<float>> inputs;
+
+    // input data shape (1, 2, 1, 3)
+    inputs.emplace_back(
+        ngraph::test::NDArray<float, 4>({{{{-1., 0., 1.}}, {{2., 3., 4.}}}}).get_vector());
+
+    // scale (3)
+    inputs.emplace_back(std::vector<float>{1., 1.5});
+    // bias (3)
+    inputs.emplace_back(std::vector<float>{0., 1.});
+    // mean (3)
+    inputs.emplace_back(std::vector<float>{0., 3});
+    // var (3)
+    inputs.emplace_back(std::vector<float>{1., 1.5});
+
+    // shape (1, 2, 1, 3)
+    auto expected_output = ngraph::test::NDArray<float, 4>(
+                               {{{{-0.999995, 0., 0.999995}}, {{-0.22474074, 1., 2.2247407}}}})
+                               .get_vector();
+
+    auto result_vectors = execute(function, inputs, "CPU");
```

Comment:
These tests should be run against all available backends, not just CPU. You should use NGRAPH_TEST instead of TEST and add this source file to the list of configured source files. Look at backend_test.in.cpp. If you want to address this later it would be ok or I can help you get this setup.